src/frontend/interfaces/page-json.interface.ts

```diff
@@ -1,3 +1,6 @@
+import { IsFunction } from 'src/backend'
```

Comment:
Sorry I haven't noticed this earlier. Can you change the import path to be relative (not start with `src/`)?